api/app/Exceptions/ErrorCode.php

```diff
@@ -19,5 +19,7 @@ final class ErrorCode
     public const USER_ALREADY_VERIFIED_EXCEPTION = 1010;
     public const EMAIL_IS_NOT_VERIFIED_EXCEPTION = 1011;
     public const WRONG_DATE_TIME_EXCEPTION = 1012;
-    public const GOOGLE_OAUTH_EXCEPTION = 1013;
```

Comment:
You must not change existing exception codes as they might be already in use.

Reply:
Where i can check if it in use?

Reply:
Client code, pull requests related to the feature, branches without pull requests which other devs are working on.

Reply:
That is not used anywhere

Reply:
It is not a reason to change it, you can not be 100% sure.